sdl_viewer/src/lib.rs

```diff
@@ -30,6 +31,11 @@ pub mod gl {
     include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
 }
 
+pub mod box_drawer;
+pub mod color;
+// TODO(tschiwietz,hrapp): Use 'Color' in the 'Point' struct in point_cloud_viewer/src/lib.rs (top level crate)
```

Comment:
1) use my GitHub handle instead of my username.
2) But do not name me here at all. The TODO author should be mentioned according to google style. I do not get why myself - the git log already contains that information.